### DISCUSSION TITLE
Query a model and sort its relations

### DISCUSSION THREAD

**Discussion by perroudsky** [prisma/prisma#6662] [Q&A] [answered]
Hi everyone !

Given this schema : 
``` psl
model User {
  id        Int        @id @default(autoincrement())
  name      String     @unique
  likes Like[]
}

model Like {
  id        String   @id @default(cuid())
  authorId  String?
  author    User?    @relation(fields: [authorId], references: [id])
  createdAt DateTime @default(now())
}
```

Can you directly query the user and sort his likes by last created ?

Thanks !

**Comment by perroudsky** [comment, marked as answer]
found my answer!
for reference:

```javascript
export const User = objectType({
  name: 'User',
  definition(t) {
      // ...
      t.nonNull.list.nonNull.field('likes', {
        type: 'Like',
        resolve: (parent, args, ctx) => {
          return ctx.prisma.user
            .findUnique({
              where: {
                id: parent.id || undefined,
              },
            })
            // HERE !
            .likes({ orderBy: { createdAt: 'desc' } })
        },
      })
  },
})
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
